src/main/scala/com/ibm/couchdb/Model.scala

```diff
@@ -73,6 +73,14 @@ case class CouchAttachment(content_type: String,
   def toBytes: Array[Byte] = Base64.getDecoder.decode(data)
 }
 
+case object CouchAttachment {
+  def fromBytes(data: Array[Byte], content_type: String = ""): CouchAttachment = {
```

Comment:
What do you think of having an overloaded method that takes in a checked content-type. Maybe we can use the http4s `Content-Type` class ? https://github.com/http4s/http4s/blob/c35545684132fe5f2091ef814419dbd048a56bb2/core/src/main/scala/org/http4s/headers/Content-Type.scala